validator/validator/validator.py

```diff
@@ -489,7 +493,7 @@ def get_score_buckets(self) -> list[WinnerList]:
         sorted_contestants = [
             (uid, score)
             for uid, score in sorted(enumerate(scores), key=lambda score: score[1])
```

Comment:
should just be the for `uid in range` here and calculating using the metric if metric is not None